client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -63,7 +63,6 @@ export const StocktakeLineEdit: FC<StocktakeLineEditProps> = ({
   const t = useTranslation();
   const { highlightRows } = useRowHighlight();
   const { error } = useNotification();
-  const { isGrouped } = useIsGrouped('stocktake');
```

Comment:
Simply calling this hook seems to make that "expand all" chevron appear in the data table headers